### PR TITLE
FEC-10909 FEC-12435 Ads progress indicator/seekbar fixes

### PR DESCRIPTION
**Pull request by chausov**
Solves:
FEC-10909 progress indication added for ads
FEC-12435 seeking disabled while ads are playing

### CODE REVIEW

File: Sources/UI/KPMediaPlayer.swift [hunk 1]
```diff
@@ -378,6 +379,32 @@ extension KPMediaPlayer {
         }
     }
     
+    private func handleAdEvents() {
```
Comment (x-NR-x) [inline Sources/UI/KPMediaPlayer.swift]: Why was this separated? The adStarted and adComplete events are already in the registerAdEvents function.
And please use the type alias KPAdEvent and not the Playkit AdEvent.

**Review by x-NR-x — CHANGES_REQUESTED** [review]

File: Sources/UI/KPMediaPlayer.swift [hunk 2]
```diff
@@ -410,8 +410,10 @@ extension KPMediaPlayer {
                     self.activityIndicator.stopAnimating()
                     self.playPauseButton.displayState = .pause
                     self.mediaProgressSlider.isEnabled = false
+                    self.mediaProgressSlider.maximumTrackTintColor = UIColor.orange
                 case is KPAdEvent.AdComplete:
                     self.mediaProgressSlider.isEnabled = true
+                    self.mediaProgressSlider.maximumTrackTintColor = UIColor.lightGray.withAlphaComponent(0.5)
```
Comment (x-NR-x) [inline Sources/UI/KPMediaPlayer.swift:425]: I think this needs to be in ad skipped as well.

**Review by x-NR-x — CHANGES_REQUESTED** [review]
Please fix:
- The duration label is not updated back to the media value after the ad (pre/mid).
- The 'mediaProgressSlider.bufferValue' should not be shown upon an ad, but should be seen after the ad.

**Comment by chausov** [comment]
> Please fix:
> 
> * The duration label is not updated back to the media value after the ad (pre/mid).
> * The 'mediaProgressSlider.bufferValue' should not be shown upon an ad, but should be seen after the ad.

Fixed

**Review by x-NR-x — APPROVED** [review]

**Pull request merged**